theano/sandbox/gpuarray/blas.py

```diff
@@ -180,10 +159,11 @@ def c_code_cache_version(self):
 
 class GpuGer(BlasOp, Ger):
     def make_node(self, A, alpha, x, y):
+        ctx_name = infer_context_name(A, x, y)
         Ger.make_node(self, A, alpha, x, y)
-        A = as_gpuarray_variable(A)
-        x = as_gpuarray_variable(x)
-        y = as_gpuarray_variable(y)
+        A = as_gpuarray_variable(A, ctx_name)
+        x = as_gpuarray_variable(x, ctx_name)
+        y = as_gpuarray_variable(y, ctx_name)
         assert A.dtype == x.dtype == y.dtype
         return Apply(self, [A, alpha, x, y], [A.type()])
 
```

Comment:
No, because these ops don't implement get_context() and so are not recognized as "context" nodes.